discovery-provider/alembic/trigger_sql/handle_challenge_disbursements.sql

```diff
@@ -0,0 +1,37 @@
+create or replace function handle_challenge_disbursement() returns trigger as $$
+declare
+  reward_manager_tx reward_manager_txs%ROWTYPE;
+begin
+
+  select * into reward_manager_tx from reward_manager_txs where reward_manager_txs.signature = new.signature limit 1;
+	
+  if reward_manager_tx is not null then
```

Comment:
not super familiar with `challenge_disbursements` and `reward_manager_txs` tables and when they get updated... but wondering about the sequence here.

In this case the notification will get created for this timeline:
* insert into `reward_manager_txs`
* insert into `challenge_disbursements` with matching signature

But is it possible the order could be reversed?
Like the `challenge_disbursements` gets created before the `reward_manager_txs` exists?

If that is possible than this trigger might fail to create the notification.

Reply:
they should be added as part of the same transaction - that's alright right